charts/datadog-operator/values.yaml

```diff
@@ -105,6 +105,14 @@ podAnnotations: {}
 # podLabels -- Allows setting additional labels for for Datadog Operator PODs
 podLabels: {}
 
+proxy:
+  # proxy.http -- Sets a proxy for all HTTP connections.
+  http: ""
+  # proxy.https -- Sets a proxy for all HTTPS connections.
+  https: ""
+  # proxy.none -- Sets networks to send traffic without a proxy.
+  none: ""
+
```

Comment:
The solution that you propose here is valid, but on the `datadog` chart, we used another approach.

To support proxy configuration threw environment variable configuration: we provide a way to configure any env-vars https://github.com/DataDog/helm-charts/blob/e167c72f8f22fcc3a0b3698690a6754e366a00cc/charts/datadog/values.yaml#L319-L324
it give a bit more flexibility.